catalog/systemd.catalog.in

```diff
@@ -364,7 +364,8 @@ Subject: Unit skipped
 Defined-By: systemd
 Support: %SUPPORT_URL%
 
-The unit @UNIT@ was skipped and has entered the 'dead' state with result '@UNIT_RESULT@'.
+The unit @UNIT@ was skipped due to an ExecCondition= command failure, and has
+entered the 'dead' state with result '@UNIT_RESULT@'.
```

Comment:
I wasn't sure that this MESSAGE_ID is only used for ExecCondition, but it seems so.

Reply:
Yup. This was suggested at https://github.com/systemd/systemd/pull/12933#discussion_r304307876 to include a reference to ExecCondition=. Thanks for merging!